docs/source/cpp/compute.rst

```diff
@@ -1707,7 +1707,7 @@ These functions select and return a subset of their input.
 +---------------+--------+--------------+--------------+--------------+-------------------------+-----------+
 | array_take    | Binary | Any          | Integer      | Input type 1 | :struct:`TakeOptions`   | \(3)      |
 +---------------+--------+--------------+--------------+--------------+-------------------------+-----------+
-| drop_null     | Unary  | Any          | -            | Input type 1 |                         | \(1)      |
+| drop_null     | Unary  | Any          |              | Input type 1 |                         | \(1)      |
```

Comment:
Referring https://github.com/apache/arrow/blob/7cc07681189e71ef986653129a6804a81d32ce8f/docs/source/cpp/compute.rst?plain=1#L213, use blank for N/A.